samples/besu/simpleasset/contracts/manageAssetAny.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.8;
+
+import "./transferInterface.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
+
+/**
+ * Hashed Timelock Contract on Fungible Assets in an Ethereum network (Support only for ERC20 tokens right now)
+ **/
+
+contract InteroperationBaseClassERCAny is ERC1155Holder {
```

Comment:
I always wanted to change the name of the class, can we use some name specific to asset-exchange?
May be: `AssetExchangeContract` or `HTLContract`, or if you have any better name you can suggest.